ansible/postgresql/init.yml

```diff
@@ -0,0 +1,279 @@
+
+    - name: init db | install uuid-ossp for qms database
+      community.postgresql.postgresql_ext:
+        login_host: "{{ pg_login_host }}"
+        port: "{{ pg_listen_port }}"
+        login_user: "{{ pg_login_user }}"
+        login_password: "{{ pg_login_password }}"
+        db: "qms"
+        name: "uuid-ossp"
+      when: install_exts is defined and install_exts and qms is defined and qms
+
+    - name: init db | install moddatetime for qms database
+      community.postgresql.postgresql_ext:
+        login_host: "{{ pg_login_host }}"
+        port: "{{ pg_listen_port }}"
+        login_user: "{{ pg_login_user }}"
+        login_password: "{{ pg_login_password }}"
+        db: "qms"
+        name: "moddatetime"
+      when: install_exts is defined and install_exts and qms is defined and qms
+
+    - name: init db | install btree_gist for qms database
+      community.postgresql.postgresql_ext:
+        login_host: "{{ pg_login_host }}"
+        port: "{{ pg_listen_port }}"
+        login_user: "{{ pg_login_user }}"
+        login_password: "{{ pg_login_password }}"
+        db: "qms"
+        name: "btree_gist"
+      when: install_exts is defined and install_exts and qms is defined and qms
+
+    - name: init_db | install insert_username for qms database
+      community.postgresql.postgresql_ext:
+        login_host: "{{ pg_login_host }}"
+        port: "{{ pg_listen_port }}"
+        login_user: "{{ pg_login_user }}"
+        login_password: "{{ pg_login_password }}"
+        db: "qms"
+        name: "insert_username"
+      when: install_exts is defined and install_exts and qms is defined and qms
+
+    - name: init db  | create unleash database
+      community.postgresql.postgresql_db:
+        login_host: "{{ pg_login_host }}"
+        port: "{{ pg_listen_port }}"
+        login_user: "{{ pg_login_user }}"
+        login_password: "{{ pg_login_password }}"
+        name: "unleash"
+        owner: "{{ _dbms_connection_user }}"
+        encoding: "{{ encoding }}"
+        lc_collate: "{{ lc_collate }}"
+        lc_ctype: "{{ lc_ctype }}"
+        template: "{{ template }}"
+      when: create_dbs is defined and create_dbs and unleash is defined and unleash
+
+    - name: init db | create staging directory
+      ansible.builtin.tempfile:
+        state: directory
+        suffix: "-dbs"
+      register: dbs_dir
+
+    # - name: init db | download migrate tarball
+    #   ansible.builtin.get_url:
+    #     url: "https://github.com/golang-migrate/migrate/releases/download/{{ migrate_release_version }}/{{ migrate_tarball_name }}"
+    #     dest: "{{ dbs_dir.path }}/{{ migrate_tarball_name }}"
+
+    - name: init db | extract migrate tarball
+      ansible.builtin.unarchive:
+        src: "https://github.com/golang-migrate/migrate/releases/download/{{ migrate_release_version }}/{{ migrate_tarball_name }}"
+        dest: "{{ dbs_dir.path }}"
+        remote_src: yes
+        creates: "{{ dbs_dir.path }}/migrate"
```

Comment:
I didn't realize that the `unarchive` task understood URLs in the `src` parameter. Nice! 👍